src/main/java/pl/szczesniak/dominik/whattowatch/users/domain/model/Username.java

```diff
@@ -0,0 +1,23 @@
+package pl.szczesniak.dominik.whattowatch.users.domain.model;
+
+
+import lombok.Value;
+import pl.szczesniak.dominik.whattowatch.users.domain.model.exceptions.InvalidUsernameException;
+
+@Value
+public class Username {
+
+	String value;
+
+	public Username(final String value) {
+		this.value = value;
+		if (!validateName(value)) {
+			throw new InvalidUsernameException("Invalid username: " + value + ". Username can have up to 25 characters and contain only letters.");
+		}
+	}
+
+	public static boolean validateName(String name) {
+		return name.matches("(?i)[a-z](.{0,23}[a-z])?");
```

Comment:
czemu nie ma testów do tego regexa? Skąd go wziąłeś i co on dokładnie sprawdza? validateName to tak trochę zbyt ogólna nazwa, nie wiem co tu dokładnie sprawdzasz

widzę, że nie zrobiłeś tego w TDD, bo byś nie miał tego regexa, prostszy by wystarczył dla tych testów co masz.